Resources/Private/Extensions/News/Partials/List/Item.html

```diff
@@ -1,4 +1,7 @@
-{namespace n=GeorgRinger\News\ViewHelpers}<!--
+{namespace n=GeorgRinger\News\ViewHelpers}
+{namespace v=KayStrobach\Themes\ViewHelpers}
```

Comment:
We have yet to update the 'Themes' extension in the git submodules type of t3kit which has this viewhelper. I have made a feature request in podio to hopefully update the 'Themes' extension.